### ISSUE TITLE
ZetaChain (EVM) is missing from the Select Chain UI

### ISSUE THREAD

**Issue by fadeev** [chainapsis/keplr-chain-registry#175]
"You can select EVM chains in the next step":

<img width="1840" alt="Screenshot 2023-07-06 at 18 41 19" src="https://github.com/chainapsis/keplr-chain-registry/assets/332151/d33a6098-6585-481f-9953-dd7760a1e8f4">

ZetaChain is missing from the list of EVM chains:

<img width="1840" alt="Screenshot 2023-07-06 at 18 41 25" src="https://github.com/chainapsis/keplr-chain-registry/assets/332151/51a07c48-8183-4d33-b5f0-15735655ac44">

**Comment by fadeev** [comment]
Looks like I had to re-install Keplr for it to show up.

**Comment by mkaczanowski** [comment]
I can't find the network even though it was added. I tried to remove and install again, and also adding new account / profile when the network was added in previous profile.

Overall, I had the same issue with other ~ethermint networks that are not enabled by default. Any idea how to fix this?

![image](https://github.com/chainapsis/keplr-chain-registry/assets/4912567/d83bfab7-94b2-49c7-9326-485bcabec6d1)
![image](https://github.com/chainapsis/keplr-chain-registry/assets/4912567/960aff47-cc4b-4a6b-9a7c-5535deb8ebf3)
